examples/tutorials/ctc_forced_aligment_api_tutorial.py

```diff
@@ -0,0 +1,683 @@
+    segment = waveform[:, x0:x1]
+    return IPython.display.Audio(segment.numpy(), rate=sample_rate)
+
+# Generate the audio for each segment
+print(transcript)
+IPython.display.Audio(SPEECH_FILE)
+
+######################################################################
+#
+
+display_segment(0, waveform, word_segments, frame_alignment)
+
+######################################################################
+#
+
+display_segment(1, waveform, word_segments, frame_alignment)
+
+######################################################################
+#
+
+display_segment(2, waveform, word_segments, frame_alignment)
+
+######################################################################
+#
+
+display_segment(3, waveform, word_segments, frame_alignment)
+
+######################################################################
+#
+
+display_segment(4, waveform, word_segments, frame_alignment)
+
+######################################################################
+#
+
+display_segment(5, waveform, word_segments, frame_alignment)
+
+######################################################################
+#
+
+display_segment(6, waveform, word_segments, frame_alignment)
+
+######################################################################
+#
+
+display_segment(7, waveform, word_segments, frame_alignment)
+
+######################################################################
+#
+
+display_segment(8, waveform, word_segments, frame_alignment)
+
+
+######################################################################
+# II. Advancd usages
+# ------------------
+#
+# Aligning non-English data
+# ~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# The provided Wav2Vec2 acoustic model was trained on data from multiple
+# languages with romanized transcripts, and is thereby capable of decoding
+# non-English data. Here we show an example of computing forced alignments
+# on a German utterance after romanizing the transcript using the UROMAN
+# tool.
+#
+
+import re
+import os
+
+
+def compute_and_plot_alignments(transcript, dictionary, emission, waveform):
+    frames, frame_alignment, _ = compute_alignments(transcript, dictionary, emission)
+    segments = merge_repeats(frames, transcript)
+    word_segments = merge_words(transcript, segments)
+    plot_alignments(
+        segments,
+        word_segments,
+        waveform[0],
+        emission.shape[0]
+    )
+    plt.show()
+    return word_segments, frame_alignment
+
+
+def normalize_uroman(text):
+    text = text.lower()
+    text = text.replace("’", "'")
+    text = re.sub("([^a-z' ])", " ", text)
+    text = re.sub(' +', ' ', text)
+    return text.strip()
+
+os.system("echo 'aber seit ich bei ihnen das brot hole brauch ich viel weniger schulze wandte sich ab die kinder taten ihm leid' > test.txt")
+os.system("git clone https://github.com/isi-nlp/uroman")
+os.system("uroman/bin/uroman.pl < test.txt > test_romanized.txt")
```

Comment:
We can't expect Perl interpreter is available always, so this needs to be replaced with something different.